documentation/scaling_puppet_server.markdown

```diff
@@ -95,28 +95,31 @@ If you [use `SRV` records for agents](#using-dns-srv-records), you can use the `
 
 To add a compile master to your deployment, begin by [installing and configuring Puppet Server](./install_from_packages.markdown) on it.
 
+In the compile master's [`puppet.conf`](https://puppet.com/docs/puppet/latest/config_file_main.html), in the `[main]` configuration block, set the [`ca_server`](https://puppet.com/docs/puppet/latest/configuration.html#caserver) setting to the hostname of the server acting as the certificate authority.
+
 Before running `puppet agent` or `puppet master` on the new server:
 
--   [Disable Puppet Server's certificate authority services](./configuration.markdown#service-bootstrapping).
+1.   [Disable Puppet Server's certificate authority services](./configuration.markdown#service-bootstrapping).
 
-    -   If you're using the [individual agent configuration method of CA centralization](#directing-individual-agents-to-a-central-ca), set `ca_server` in `puppet.conf` to the hostname of your CA server in the `[main]` config block.
-    -   If an `ssldir` is configured, make sure it's configured in the `[main]` block only.
+     If you're using the [individual agent configuration method of CA centralization](#directing-individual-agents-to-a-central-ca), set `ca_server` in `puppet.conf` to the hostname of your CA server in the `[main]` config block.
+    
+    If an `ssldir` is configured, make sure it's configured in the `[main]` block only.
 
--   If you're using the [DNS round robin method](#using-round-robin-dns) of agent load balancing, or a [load balancer](#using-a-load-balancer) in TCP proxying mode, provide compile masters with certificates using DNS Subject Alternative Names.
+2.   If you're using the [DNS round robin method](#using-round-robin-dns) of agent load balancing, or a [load balancer](#using-a-load-balancer) in TCP proxying mode, provide compile masters with certificates using DNS Subject Alternative Names.
 
-    -   Configure `dns_alt_names` in the `[main]` block of `puppet.conf` to cover every DNS name that might be used by an agent to access this master.
+     Configure `dns_alt_names` in the `[main]` block of `puppet.conf` to cover every DNS name that might be used by an agent to access this master.
 
         ```
         dns_alt_names = puppet,puppet.example.com,puppet.site-a.example.com
         ```
 
-    -   If the agent or master has been run and already created a certificate, remove it by running `sudo rm -r $(puppet master --configprint ssldir)`. If an agent has requested a certificate from the master, delete it there to re-issue a new one with the alt names: `puppetserver ca clean master-2.example.com`.
+     If the agent or master has been run and already created a certificate, remove it by running `sudo rm -r $(puppet master --configprint ssldir)`. If an agent has requested a certificate from the master, delete it there to re-issue a new one with the alt names: `puppetserver ca clean master-2.example.com`.
```

Comment:
Should this say to use `puppet ssl clean` now instead of `rm -rf`?

Reply:
done!